cmd/subctl/cloud.go

```diff
@@ -57,8 +60,13 @@ func init() {
 	restConfigProducer.AddKubeContextFlag(cloudCmd)
 	rootCmd.AddCommand(cloudCmd)
 
-	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.Natt, "natt-port", 4500, "IPSec NAT traversal port")
-	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.NatDiscovery, "nat-discovery-port", 4490, "NAT discovery port")
+	defaultTunnelPort, _ := strconv.ParseInt(subv1.DefaultUDPPort, 0, 32)
+	defaultNatDiscoveryPort, _ := strconv.ParseInt(subv1.DefaultNATTDiscoveryPort, 0, 32)
+
+	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.Natt, "natt-port", uint16(defaultTunnelPort),
+		"IPSec NAT traversal port")
+	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.NatDiscovery, "nat-discovery-port",
+		uint16(defaultNatDiscoveryPort), "NAT discovery port")
 	cloudPrepareCmd.PersistentFlags().Uint16Var(&cloudPorts.Vxlan, "vxlan-port", 4800, "Internal VXLAN port")
```

Comment:
I tried to use constants for vxlan-port but when i include the route-agent/kubeproxy constants file, its trying to import many other dependencies and is throwing errors. So, avoided making the change.